kyuubi-common/src/main/scala/org/apache/kyuubi/operation/AbstractOperation.scala

```diff
@@ -162,3 +162,9 @@ abstract class AbstractOperation(opType: OperationType, session: Session)
     }
   }
 }
+
+object AbstractOperation {
```

Comment:
why an object is named abstract?

Reply:
e... forget to change the name.
Is ok to named OperationFieldKey in this class?